docs/pages/brownfield/lifecycle-listeners.mdx

```diff
@@ -0,0 +1,116 @@
+  subscribers. Modules can provide an `ExpoAppDelegateSubscriber` implementation to register
+  callbacks.
+- <PlatformTag platform="android" /> `ApplicationLifecycleDispatcher` and `ReactActivityHandler`
+  forward `Application` and `Activity` lifecycle events to registered listeners. Modules can provide
+  `ReactActivityLifecycleListener` and `ApplicationLifecycleListener` implementations through a
+  `Package` class to register callbacks.
+
+Using these mechanisms allows modules to register behavior without requiring you to edit native entry points repeatedly.
+
+## Configure your native project
+
+### Android
+
+To integrate `Application` lifecycle listeners on Android, forward the `onCreate()` and `onConfigurationChanged()` calls from your `Application` class to `ApplicationLifecycleDispatcher`:
+
+```diff
+class MainApplication : Application() {
+
+  override fun onCreate() {
+    super.onCreate()
+    ...
++   ApplicationLifecycleDispatcher.onApplicationCreate(this)
+  }
+
+  override fun onConfigurationChanged(newConfig: Configuration) {
+    super.onConfigurationChanged(newConfig)
+    ...
++   ApplicationLifecycleDispatcher.onConfigurationChanged(this, newConfig)
+  }
+}
+```
+
+### iOS
+
+To integrate `AppDelegate` subscribers on iOS, forward the relevant calls to `ExpoAppDelegateSubscriberManager` in your existing `AppDelegate` implementation so that subscribers can respond to them:
+
+```diff AppDelegate.swift
++import Expo
+
+public class AppDelegate: UIApplicationDelegate {
+
+  open func application(
+    _ application: UIApplication,
+    willFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil
+  ) -> Bool {
++    return ExpoAppDelegateSubscriberManager.application(application, willFinishLaunchingWithOptions: launchOptions)
+  }
+
+  open func application(
+    _ application: UIApplication,
+    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil
+  ) -> Bool {
++    return ExpoAppDelegateSubscriberManager.application(application, didFinishLaunchingWithOptions: launchOptions)
+  }
+
+  ...
+}
+```
+
+Alternatively, if your `AppDelegate` doesn't already extend another class, you can simplify the setup by inheriting from `ExpoAppDelegate`, which handles the forwarding automatically:
+
+```diff AppDelegate.swift
++import Expo
+
+@main
+-public class AppDelegate: NSObject, UIApplicationDelegate {
++public class AppDelegate: ExpoAppDelegate {
+```
+
+> **Note:** Not all `UIApplicationDelegate` methods that could cause significant side effects are supported. See the Expo source (**ExpoAppDelegate.swift**) for the full list of forwarded methods if you need to rely on a specific delegate.
+
+## Test your integration
+
+A simple way to test if the callbacks are working correctly is to install a module that relies on them. You can install `expo-linking`, which uses lifecycle listeners to handle deep links:
```

Comment:
```suggestion
To test if the callbacks are working correctly, install a module that relies on them. Install `expo-linking`, which uses lifecycle listeners to handle deep links:
```

I'd avoid using terms like "simple" since that is user should be left to the reader's perspective.